jetty-core/jetty-quic/jetty-quic-quiche/jetty-quic-quiche-jna/src/main/java/org/eclipse/jetty/quic/quiche/jna/JnaQuicheBinding.java

```diff
@@ -21,26 +21,25 @@
 import org.eclipse.jetty.quic.quiche.QuicheBinding;
 import org.eclipse.jetty.quic.quiche.QuicheConfig;
 import org.eclipse.jetty.quic.quiche.QuicheConnection;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class JnaQuicheBinding implements QuicheBinding
 {
-    private static final Logger LOG = LoggerFactory.getLogger(JnaQuicheBinding.class);
-
     @Override
-    public boolean isUsable()
+    public Throwable checkUsable()
     {
         try
         {
-            // Make a Quiche call to confirm.
-            LibQuiche.INSTANCE.quiche_version();
-            return true;
+            LibQuiche.initialize();
+            return null;
+        }
+        catch (ExceptionInInitializerError e)
+        {
+            Throwable cause = e.getCause();
+            return cause != null ? cause : e;
         }
         catch (Throwable x)
         {
-            LOG.debug("JNA quiche binding is not usable", x);
-            return false;
+            return x;
```

Comment:
Store the exception to be used in `toString()`.

Reply:
Done.